definitions/ext-flow_device/summary_metrics.yml

```diff
@@ -0,0 +1,33 @@
+endpoints:
+  title: Endpoints
+  unit: COUNT
+  query:
+    select: uniqueCount(ip_address)
+    from: Metric
+    where: "metricName LIKE 'kentik.rollup.bytes.%'"
+    eventId: entity.guid
+
+conversations:
+  title: Conversations
+  unit: COUNT
+  query:
+    select: uniqueCount(tuple(ip_address,port))
```

Comment:
unsure if this will work out of the box in summaries. we should check

Reply:
it seems to be working right now in the [ext-vpc_network](https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/b972d23d8ab26cec93dfc77a0ea5ae8aacbeb3fd/definitions/ext-vpc_network/summary_metrics.yml#L9-L16) entities that are live